### PR TITLE
use slow

### LINKED ISSUE #521: Slow network mobile menu not always working
## Are you reporting a bug or a feature?

 - [x] Bug
 - [ ] Feature

## Expected Behavior

Mobile burger button should always open/close mobile naviagtion.

## Actual Behavior

Does not always work on slow networks.

## Steps to Reproduce the Problem

1.
2.
3.

## Specifications

 - Browser: iOS Safari
 - Browser Version: 11
 - Node Version:
 - NPM Version:
 - Platform:

### PR DESCRIPTION
**Pull request by AndyOGo**
Fixes #521
Mitigates #26 

Changes proposed in this pull request:

 - adds shared var to window

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
 
 # How Has This Been Tested?
 
 
 
 # Checklist:
 
 - [ ] My code follows the style guidelines of this project
 - [ ] I have performed a self-review of my own code
 - [ ] I have commented my code, particularly in hard-to-understand areas
 - [ ] I have made corresponding changes to the documentation
 - [ ] My changes generate no new warnings
 - [ ] I have added tests that prove my fix is effective or that my feature works
 - [ ] New and existing unit tests pass locally with my changes
 - [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by AndyOGo** [comment]
`contextNode` is undefined 😕

File: stack/server.js [hunk 1]
```diff
@@ -3,11 +3,14 @@ const constants = require('./constants');
 const express = require('express');
 const readline = require('readline');
 const compression = require('compression');
+const slow = require('connect-slow');
```
Comment (AndyOGo) [inline stack/server.js]: @LucaMele 
We could either remove `connect-slow` or keep it for future testing, but disable it for prod?

File: src/js/pubsub.js [hunk 2]
```diff
@@ -4,7 +4,8 @@ import debounce from './debounce';
 // import maybe, { toEqual } from './maybe';
 
 // @TODO: this local variable isn't shared between redundant module instance
-const subscriptions = {};
+window.__subscriptions = window.__subscriptions || {};
```
Comment (AndyOGo) [inline src/js/pubsub.js]: We really need to fix this!
#26

Comment (LucaMele) [reply src/js/pubsub.js]: :) yepp along with soooo many stuff...

File: stack/server.js (hunk 1 above)
Comment (LucaMele) [reply stack/server.js]: I would comment it out..

File: stack/server.js [hunk 3]
```diff
@@ -3,11 +3,14 @@ const constants = require('./constants');
...
 
 const ENV = process.env.NODE_ENV; // second element is the first argument.
 
 const app = express();
 
+app.use(slow());
+
```
Comment (LucaMele) [inline stack/server.js]: @AndyOGo please comment it out instead

Comment (AndyOGo) [reply stack/server.js]: done

File: src/js/pubsub.js (hunk 2 above)
Comment (AndyOGo) [reply src/js/pubsub.js]: true

**Pull request merged**